src/lib/zexe_backend/zexe_backend_common/endoscale_round.ml

```diff
@@ -0,0 +1,49 @@
+module Endoscale_round = struct
+  open Core_kernel
+  module H_list = Snarky_backendless.H_list
+
+  [%%versioned
+    module Stable = struct
+      module V1 = struct
+        type 'a t =
+          {
+            b2i1: 'a; xt: 'a;
+            b2i: 'a; xq: 'a; yt: 'a; 
+            xp: 'a; l1: 'a; yp: 'a; 
+            xs: 'a; ys: 'a
+          } [@@deriving sexp, fields]
+      end
+    end]
+
+  type 'a t = 'a Stable.Latest.t =
```

Comment:
this type is generated from `%%versioned`. Think you'll need to add `hlist` to the `V1` deriving list.

Reply:
Added, thank you